R/execution.R

```diff
@@ -203,6 +206,7 @@ executeDqChecks <- function(connectionDetails,
   options(scipen = 999)
 
   # capture metadata -----------------------------------------------------------------------
+if (!sqlOnly) {
```

Comment:
If running in  `sqlOnly` mode, we still want to create a metadata object and add the DQD version, like so:
```suggestion
  metadata <- list()
  metadata$DQD_VERSION <- as.character(packageVersion("DataQualityDashboard"))
  if (!sqlOnly) {
```